### DISCUSSION TITLE
gltf model not loading into cache

### DISCUSSION THREAD

**Discussion by FehmiCan16** [google/model-viewer#3890] [Q&A]
Hello. Once I load my gltf model, I refresh the page and it reloads the model completely. Is there a way to cache it once the model is loaded and quickly load the model from the cache when the page is refreshed?

**Comment by elalish** [comment]
I believe if you want to store something across refreshes, you'll want to make use of `window.localStorage`. I think the browser's network cache should also be doing some of this automatically for you?
